src/block/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
maybe better add ul in reset.scss

Reply:
agree, it's better to move this code in reset.scss